vspace/src/main/webapp/WEB-INF/tiles/skeleton_bt4.jsp

```diff
@@ -93,7 +93,7 @@
         </main>
       </div>
     </div>
-    
+    <tiles:insertAttribute name="deleteModal" />
```

Comment:
this would make a modal delete attribute available in all pages, which is unnecessary and doesn't make sense. It just blows up the list of attributes. put it where it is actually needed.

Reply:
if you don't want to put the code inside the page itself, you can always include it directly in the jsp page via an include.